Yoast/javascript
# P2-213 add property validation

## Summary


This PR can be summarized in the following changelog entry:

* [schema-blocks] Adds basic property validation to the blocks.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

Not really testable yet this early in development.

* Run the unit tests using `yarn test --coverage`.
* See that the added code is covered.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * It does not impact any user-facing functionality.

## UI changes

## Quality assurance


Fixes #
